framework/test/CMakeLists.txt

```diff
@@ -107,18 +107,7 @@ set_property(TARGET ${_test_driver} PROPERTY US_BUNDLE_NAME main)
 if (("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang") OR
     ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "AppleClang") OR
     ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU"))
-  #target_compile_optinons(${_test_driver} PUBLIC "-Wno-deprecated-declarations")
-  set_source_files_properties(BundleActivatorTest.cpp PROPERTIES COMPILE_FLAGS -Wno-deprecated-declarations)
-  set_source_files_properties(BundleHooksTest.cpp PROPERTIES COMPILE_FLAGS -Wno-deprecated-declarations)
-  set_source_files_properties(BundleTest.cpp PROPERTIES COMPILE_FLAGS -Wno-deprecated-declarations)
-  set_source_files_properties(ConcurrencyTest.cpp PROPERTIES COMPILE_FLAGS -Wno-deprecated-declarations)
-  set_source_files_properties(FrameworkListenerTest.cpp PROPERTIES COMPILE_FLAGS -Wno-deprecated-declarations)
-  set_source_files_properties(FrameworkTest.cpp PROPERTIES COMPILE_FLAGS -Wno-deprecated-declarations)
-  set_source_files_properties(MultipleListenersTest.cpp PROPERTIES COMPILE_FLAGS -Wno-deprecated-declarations)
-  set_source_files_properties(ServiceListenerTest.cpp PROPERTIES COMPILE_FLAGS -Wno-deprecated-declarations)
-  set_source_files_properties(ServiceHooksTest.cpp PROPERTIES COMPILE_FLAGS -Wno-deprecated-declarations)
-  set_source_files_properties(ServiceRegistryPerformanceTest.cpp PROPERTIES COMPILE_FLAGS -Wno-deprecated-declarations)
-  set_source_files_properties(StaticBundleTest PROPERTIES COMPILE_FLAGS -Wno-deprecated-declarations)
+  set_source_files_properties(${_srcs} PROPERTIES COMPILE_FLAGS -Wno-deprecated-declarations)
```

Comment:
Did `-Wno-error=deprecated-declarations` not work, or do think suppressing the warnings completely is preferable?